### PR TITLE
3444 - 2

### PR DESCRIPTION
**Pull request by svintbel**
## Номер
3444

## Номер задания
2

## Ссылка на видео с демо
https://youtu.be/XPdhuyNertU

## Комментарии
simple realization

### CODE REVIEW

File: 3444/2/commands/check_in.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+module CheckIn
+  include PathHelper
+
+  def checkin!(*)
+    if registred?
+      enter_camp_time
+      save_context :checkin_photo
+      respond_with :message, text: 'You should send me a selfie'
+    else
+      respond_with :message, text: 'Please, authorize befores'
```
Comment (anatoliliotych) [inline 3444/2/commands/check_in.rb]: typo in before

File: 3444/2/commands/start.rb [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# rubocop:disable Style/GlobalVars
```
Comment (anatoliliotych) [inline 3444/2/commands/start.rb]: remove this

File: 3444/2/.env [hunk 3]
```diff
@@ -0,0 +1 @@
+token=
```
Comment (anatoliliotych) [inline 3444/2/.env]: Never commit .env file. Use `.env.sample`

File: 3444/2/.gitignore [hunk 4]
```diff
@@ -0,0 +1,2 @@
+.env
```
Comment (anatoliliotych) [inline 3444/2/.gitignore]: how it's possible to have it in git repo and in gitignore? :)

Comment (svintbel) [reply 3444/2/.gitignore]: it's possible cause i did commit with .env, then founded it in index and decided to add it into gitignore, after that made rebase)

Comment (anatoliliotych) [reply 3444/2/.gitignore]: It doesn't make sense to have them both.

Comment (anatoliliotych) [reply 3444/2/.gitignore]: read my comments about having .env.sample

Comment (svintbel) [reply 3444/2/.gitignore]: done

**Pull request merged**